core/urls.py

```diff
@@ -16,12 +16,12 @@
 """
 from django.contrib import admin
 from django.urls import path, include
-
 from django.conf import settings
 from django.conf.urls.static import static
 
 urlpatterns = [
     path('admin/', admin.site.urls),
     path('api/users/', include('users.urls')),
     path('api/projects/', include('projects.urls')),
+    path('api/fastapi/', include('fastapi_app.main')),
```

Comment:
Including FastAPI routes in Django's urlpatterns will not work as expected because FastAPI does not provide Django-compatible URL patterns. The FastAPI app should be mounted at the ASGI level (as you did in `asgi.py`), not in `urls.py`. Please remove this line to avoid confusion and potential errors.